pkg/api/resource_helpers.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
This file contains methods on structs defined in this package. That is the reason for placing them in 'package api'